generatedDockerfiles/Dockerfile.centos7-devel

```diff
@@ -212,6 +217,7 @@ CMD [ "/bin/bash" ]
 RUN source activate rapids \
     && env \
     && conda list \
+    && conda uninstall -y blas openblas \
```

Comment:
Blas is coming from previous commit. It looks like dockerfiles were not regenerated for blas patch and they show up now

Reply:
Hopefully they'll get updated. Can you put a FIXME comment saying this should also be temporary?

Reply:
Fixed